js/map-content-to-topper.js

```diff
@@ -0,0 +1,162 @@
+const themeImageRatio = {
```

Comment:
@notlee i've tried to remove the next-specific things from in here but it's still a little baroque. i can clean it up further if needed

Reply:
Looks good to me -- easy to digest at least. Up to you. :)